crowdin.yaml

```diff
@@ -43,6 +43,10 @@ files:
         'vi': 'vi'
         'zh-CN': 'zh-Hans'
         'zh-TW': 'zh-Hant'
+  -
```

Comment:
I think this will fix the langs according to https://docusaurus.io/docs/en/translation.html#versioned-translations